src/fragments/lib-v1/storage/js/download.mdx

```diff
@@ -45,21 +46,22 @@ import { getUrl } from 'aws-amplify/storage';
 
 // To check for existence of a file
 await getUrl({
-    key: filename,
-    options: {
-      validateObjectExistence: true,  // defaults to false
-    },
+	key: filename,
+	options: {
+		validateObjectExistence: true, // defaults to false
+	},
 });
+
 ```
 
 ### getUrl expiry
 
 You can use `expiresIn` option to limit the availability of your URLs. This configuration returns the pre-signed URL that expires in 60 seconds:
 
-```javascript
+```typescript
```

Comment:
Have we agreed to use typescripts in the examples? It would be a big change. If we haven't agreed on the change, let's stick to the js